lambda_functions/v1/tests/opg_sirius_service/test_build_sirius_url.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+import hypothesis.strategies as st
+import pytest
+import validators
+from hypothesis import given, settings, example, HealthCheck
+from .conftest import (
+    max_examples,
+    test_sirius_service,
+)
+
+
+@given(
+    endpoint=st.text(), url_params=st.dictionaries(st.text(), st.text()),
+)
+@example(
+    endpoint="v1/api/public/documents",
```

Comment:
Only thing on this is that the url example is one for the reports documents endpoint repo, so might be confusing in the lpa data repo?

Reply:
Good point, I've switched that to an LPA endpoint now 👍🏽